app/components/verkiezingen/burgemeester-alert.js

```diff
@@ -50,11 +34,9 @@ export default class VerkiezingenBcsdVoorzitterAlertComponent extends Component
     } else if (aangewezenBurgemeesters.length > 1) {
       this.errorMessage = `Er moet exact één aangewezen burgemeester zijn. Er werden er ${aangewezenBurgemeesters.length} gevonden.`;
     } else {
-      this.errorMessage = '';
+      this.errorMessage = null;
     }
-
-    await timeout(5000);
-    this.handleErrorMessage.perform();
```

Comment:
this one was polling is that no longer necessary now?

Reply:
Because of the trigger it is not (forceCompute) but i think we are "mishandelen" it